python/ngen_cal/src/ngen/cal/calibration_set.py

```diff
@@ -134,11 +113,11 @@ class UniformCalibrationSet(CalibrationSet, Adjustable):
         A HY_Features based catchment with additional calibration information/functionality
     """
 
-    def __init__(self, eval_nexus: Nexus, routing_output: 'Path', start_time: str, end_time: str, eval_params: 'EvaluationOptions', params: dict = {}):
+    def __init__(self, eval_nexus: Nexus, hooks: 'HookRelay', start_time: str, end_time: str, eval_params: 'EvaluationOptions', params: dict = {}):
```

Comment:
Same here, why not just take a `ModelHooks`?

Reply:
changed in 2602282227192d6b4703fde0cf155c1a2c003a75